common/tokensParser.go

```diff
@@ -5,8 +5,19 @@ import (
 	"strings"
 )
 
-// esdtTickerNumRandChars represents the number of hex-encoded random characters sequence of a ticker
-const esdtTickerNumRandChars = 6
+const (
+	// esdtTickerNumRandChars represents the number of hex-encoded random characters sequence of a ticker
+	esdtTickerNumRandChars = 6
+
+	// separatorChar represents the character that separated the token ticker by the random sequence
+	separatorChar = "-"
+
+	// minLengthForTickerName represents the minimum number of characters a token's ticker can have
+	minLengthForTickerName = 3
```

Comment:
do we have these consts in the vm-common package?

Reply:
unfortunately, no. we don't have exported constants for these